smoke-test/tests/deprecation/deprecation_test.py

```diff
@@ -85,30 +84,30 @@ def test_update_deprecation_all_fields(frontend_session):
     assert res_data["data"]
     assert res_data["data"]["dataset"]
     assert res_data["data"]["dataset"]["deprecation"] == {
-      'deprecated': True,
-      'decommissionTime': 0,
-      'note': 'My test note',
-      'actor': 'urn:li:corpuser:datahub'
+        "deprecated": True,
+        "decommissionTime": 0,
+        "note": "My test note",
+        "actor": "urn:li:corpuser:datahub",
     }
 
-@pytest.mark.dependency(depends=["test_healthchecks", "test_update_deprecation_all_fields"])
+
+@pytest.mark.dependency(
+    depends=["test_healthchecks", "test_update_deprecation_all_fields"]
```

Comment:
Why does this test depend on test_update_deprecation_all_fields

Reply:
I just changed  ENDPOINT -> method call in most files. Only real change is e2e and utils file. I have not looked at all dependencies for now. This got affected because lint happened so not sure about this specific one.